content-repo/extra-docs/packs/Malware_Investigation_&_Response.md

```diff
@@ -28,6 +28,9 @@ The pack closely maps evidence to MITRE ATT&CK to uncover evidence of:
 * Mirrors incidents between the EDR and Cortex XSOAR and enables incident closure with the EDR.
 * Supports aggregating incidents from EDRs, including multiple alerts and files.
 * Supports fetching malware incidents either from the EDR product or from a SIEM solution.
+* Enriches information on the associated account with the incident/alert.
```

Comment:
what about the user enrichment?

Reply:
That's the user enrichment - do you prefer rephrasing this?